tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,24 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result == True
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory ==[item_d, item_e, item_c]
```

Comment:
Tiny typo that looks like it got copy/pasted to other tests below:
```suggestion
    assert jesse.inventory == [item_d, item_e, item_c]
```